rules/github-scrum-workflow/rule.md

```diff
@@ -0,0 +1,86 @@
+---
+type: rule
+archivedreason:
+title: How to set up GitHub Scrum workflow
+guid: e63da194-c5d2-4830-9106-d164aafcddb5
+uri: do-you-know-the-how-to-set-up-github-workflow
```

Comment:
I think the uri should say "do you know how" instead of "do you know the how"

Reply:
> I think the uri should say "do you know how" instead of "do you know the how"

Done